Makefile.am

```diff
@@ -116,7 +116,8 @@ setup.h: $(srcdir)/setup.h.in Makefile
 	     -e 's|@prefix[@]|$(prefix)|g' \
 	     -e 's|@p4includedir[@]|$(p4includedir)|g' \
 	     -e 's|@p4_14includedir[@]|$(p4_14includedir)|g' \
-	$< >$@
+	$< >$@.tmp
+	@if cmp $@ $@.tmp; then rm $@.tmp; else mv $@.tmp $@; fi
```

Comment:
If you don't update setup.h then this command will run every time, by the way - i.e., setup.h will remain perpetually out of date. Given that all that happens for this target is a call to `sed`, that's not so bad, but it's worth keeping in mind.